packages/components/Switch/src/index.ts

```diff
@@ -0,0 +1 @@
+export { Switch } from './Switch';
```

Comment:
You probably want to export the types for the props/tokens/slots and name of the Switch as well. It will also be useful to export the useSwitch function for clients to access.